src/lib/utils.ts

```diff
@@ -279,3 +281,40 @@ export function getErrorLocationPath(location: any): string {
 
   return paths.join(".");
 }
+
+export function isMutationRequest(interceptMessage: CallOptions): boolean {
```

Comment:
Could do something more like

```javascript
export function isMutationRequest(interceptMessage: CallOptions): boolean {
  const path = interceptMessage?.method_definition?.path
  if (path && NON_MUTABLE_METHOD_PREFIXES.indexOf(path) > -1) {
        return false
  }
  return true
}
```

Reply:
This isn't actually the same. I'm doing a partial substring check for any `NON_MUTABLE_METHOD_PREFIXES` within a method path e.g. `Search` within `/google.ads.googleads.v2.services.GoogleAdsService/Search`. 

In your suggestion, it's a complete string match against an array, which doesn't make sense here e.g.
```javascript
["Search", "Get", "List"].indexOf("/google.ads.googleads.v2.services.GoogleAdsService/Search")
```

Reply:
Gotcha, fair enough